apps/src/templates/sectionProgress/progressTables/progressTableHelpers.js

```diff
@@ -111,33 +138,33 @@ function formatTimeSpent(studentProgress) {
     const minutes = studentProgress.timeSpent / 60;
     return `${Math.ceil(minutes)}`;
   }
-  return missingDataFormatter(studentProgress, 'timeSpent');
+  return missingDataFormatter(!!studentProgress);
```

Comment:
Is it possible that the time spent = 25 seconds which will round down to 0 minutes, and then we display the missing data format instead of 0 because 0 is falsy?

Reply:
if time spent = 25 seconds we will return at line 139.